ide/static/css/searchbar_style.css

```diff
@@ -16,8 +17,8 @@
 
 #layer-search-input {
 	position: absolute;
-	top: 36px;
-	left: 20px;
+	top: 2px;
```

Comment:
Rather than hardcoding the values you can assign relative values.

Reply:
@ZeroZeroJedenJeden try using relative values to see if that solves the layout bug